engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/consumerrecord/ConsumerRecordDeserializationSchemaFactory.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.engine.kafka.consumerrecord
+
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.streaming.connectors.kafka.KafkaDeserializationSchema
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import pl.touk.nussknacker.engine.kafka.KafkaRecordHelper
+
+import scala.reflect.classTag
+
+object ConsumerRecordDeserializationSchemaFactory {
```

Comment:
How you will create such a factory for avro? Shouldn't we have some factory method for `value: KafkaDeserializationSchema` and for key+value combination? Maybe instead of both two - IMO it is more kafka way. WDYT?